lib/radiator_web/controllers/api/audio_controller.ex

```diff
@@ -3,21 +3,33 @@ defmodule RadiatorWeb.Api.AudioController do
 
   alias Radiator.Directory.Editor
 
-  def create(conn, %{"network_id" => network_id, "audio" => params}) do
+  def create(conn, params = %{"network_id" => network_id}) do
```

Comment:
Either documentation is wrong, or you are matching wrong. (the "network_id" key is still inside an "audio")

Reply:
These (`network_id` and `episode_id`) are not params supplied but the client but the ones extracted from the route. Comes from the nesting under network/episode resource.

Reply:
Ah, some magic.